src/components/ReportTransaction.js

```diff
@@ -30,11 +30,15 @@ const propTypes = {
     /** Type of the reject transaction button */
     rejectButtonType: PropTypes.oneOf([CONST.IOU.REPORT_ACTION_TYPE.DECLINE, CONST.IOU.REPORT_ACTION_TYPE.CANCEL]).isRequired,
 
+    /** Indicates whether the pressing reject button we should hide the details sidebar */
```

Comment:
NAB, just a suggestion to improve the comment's clarity
```suggestion
    /** Indicates whether pressing the reject button should hide the details sidebar */
```

Reply:
@amyevans comment updated